### PR TITLE
improve test coverage

### PR DESCRIPTION
**Pull request by AllenZMC**
**Please provide a description of this PR:**
improve test coverage

### CODE REVIEW

**Comment by AllenZMC** [comment]
cc @ericvn

File: security/pkg/k8s/configutil_test.go [hunk 1]
```diff
@@ -306,3 +306,71 @@ func createFakeLister(kubeClient *fake.Clientset) informersv1.ConfigMapInformer
...
+			},
+			want: true,
+		},
+		{
+			name: "changed",
+			args: args{
+				cm:   createConfigMap(namespaceName, configMapName, nil),
+				data: nil,
+			},
+			want: true,
```
Comment (howardjohn) [inline security/pkg/k8s/configutil_test.go:366]: This should be checking the resulting map as well. I would encourage you to focus on getting the most coverage on tests - not just lines of code coverage, but really testing the functionality.

Comment (AllenZMC) [reply security/pkg/k8s/configutil_test.go]: @howardjohn  
Fixed.
Understood, thanks.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**